### PR TITLE
Add iron ore and steel shipping related technologies.

### PR DESCRIPTION
**Pull request by euronion**
See release notes.

## Checklist

- [n/a] Code changes are sufficiently documented; i.e. new functions contain docstrings and further explanations may be given in `doc`.
- [x] Data source for new technologies is cleary stated.
- [n/a] Newly introduced dependencies are added to `environment.yaml` (if applicable).
- [x] A note for the release notes `doc/release_notes.rst` of the upcoming release is included.
- [x] I consent to the release of this PR's code under the GPLv3 license.

### CODE REVIEW

File: inputs/manual_input.csv [hunk 1]
```diff
@@ -256,3 +256,21 @@ Haber-Bosch,hydrogen-input,0,1.1484,MWh_H2/MWh_NH3,-,"DECHEMA 2017: DECHEMA: Low
...
+electric arc furnace,electricity-input,2020,0.6395,MWh_el/t_steel,2020,"Mission Possible Partnership (2022): Steel Model (https://github.com/missionpossiblepartnership/mpp-steel-model/blob/9eca52db92bd2d9715f30e98ccaaf36677fdb516/mppsteel/data/import_data/Technology%20Business%20Cases.csv, accessed: 2022-12-03).",Based on process ‘EAF’. 
+electric arc furnace,hbi-input,2020,1,t_hbi/t_steel,2020,-,Assume HBI instead of scrap as input.Scrap would require higher input (in tonnes) as steel content is lower.
+shipping fuel methanol,fuel,2020,72,EUR/MWh_th,2020,"Based on (source 1) Hampp et al (2022), https://arxiv.org/abs/2107.01092, and (source 2): https://www.methanol.org/methanol-price-supply-demand/; both accessed: 2022-12-03.",400 EUR/t assuming range roughly in the long-term range for green methanol (source 1) and late 2020+beyond values for grey methanol (source 2).
+shipping fuel methanol,CO2 intensity,2020,0.2482,t_CO2/MWh_th,2020,-,Based on stochiometric composition.
```
Comment (fneum) [inline inputs/manual_input.csv:275]:
```suggestion
methanol,fuel,2020,72,EUR/MWh_th,2020,"Based on (source 1) Hampp et al (2022), https://arxiv.org/abs/2107.01092, and (source 2): https://www.methanol.org/methanol-price-supply-demand/; both accessed: 2022-12-03.",400 EUR/t assuming range roughly in the long-term range for green methanol (source 1) and late 2020+beyond values for grey methanol (source 2).
methanol,CO2 intensity,2020,0.2482,t_CO2/MWh_th,2020,-,Based on stochiometric composition.
```

**Review by fneum — CHANGES_REQUESTED** [review]
Can I rename "shipping fuel methanol" to more general "methanol"?

File: inputs/manual_input.csv (hunk 1 above)
Comment (euronion) [reply inputs/manual_input.csv:275]: I don't know. It would be a bit biased, but should be fine I guess?

Comment (euronion) [reply inputs/manual_input.csv:275]: Would also need to rename them in release notes.

**Pull request merged**